Podcast/ShareEndpointRequests.swift

```diff
@@ -1,43 +1,43 @@
 //
-//  FetchSharesEndpointRequest.swift
+//  ShareEndpointRequest.swift
 //  Podcast
 //
-//  Created by Natasha Armbrust on 3/4/18.
+//  Created by Jack Thompson on 8/26/18.
 //  Copyright © 2018 Cornell App Development. All rights reserved.
 //
 
 import Foundation
 import SwiftyJSON
 
 class FetchSharesEndpointRequest: EndpointRequest {
-
+    
     var offset: Int
     var max: Int
-
+    
     init(offset: Int, max: Int) {
         self.offset = offset
         self.max = max
         super.init()
-
+        
```

Comment:
Please delete this extra whitespace, it occurs a lot in this file

Reply:
Swiftlint will automatically remove white space. You guys should install it!